app/javascript/gobierto_data/webapp/components/Dataset.vue

```diff
@@ -366,9 +366,6 @@ export default {
         this.isQueryModified = true;
       }
 
-      // set the new query, trimming it to remove potentially harmful voids
-      this.currentQuery = sql.trim();
```

Comment:
I've had to remove this method. Trim removes whitespace, and this causes the query to updated and resend to the editor but without whitespaces, which causes strange and annoying cursor movement in the editor.